### PR TITLE
Remove unnecesary imports

### PR DESCRIPTION
**Pull request by javiercn**

### CODE REVIEW

File: src/Microsoft.AspNetCore.Authentication.Facebook/project.json [hunk 1]
```diff
@@ -26,9 +26,6 @@
   "frameworks": {
     "net451": {},
     "netstandard1.3": {
-      "imports": [
-        "portable-net451+win8"
-      ]
     }
```
Comment (PinpointTownes) [inline src/Microsoft.AspNetCore.Authentication.Facebook/project.json]: Should closing braces be on the same line as the opening braces for consistency with `net451`?

**Comment by Tratcher** [comment]
:shipit:

**Comment by PinpointTownes** [comment]
Before merging, you should determine why a really old JSON.NET version is (indirectly) referenced (which explains why the build failed):

```
error: Package Newtonsoft.Json 7.0.1 is not compatible with netcoreapp1.0 (.NETCoreApp,Version=v1.0). Package Newtonsoft.Json 7.0.1 supports:
error:   - net20 (.NETFramework,Version=v2.0)
error:   - net35 (.NETFramework,Version=v3.5)
error:   - net40 (.NETFramework,Version=v4.0)
error:   - net45 (.NETFramework,Version=v4.5)
error:   - portable-dnxcore50+net45+win8+wp8+wpa81 (.NETPortable,Version=v0.0,Profile=net45+wp80+win8+wpa81+dnxcore50)
error:   - portable-net40+sl5+win8+wp8+wpa81 (.NETPortable,Version=v0.0,Profile=Profile328)
```

**Comment by Tratcher** [comment]
image

**Comment by javiercn** [comment]
We took a new CLI this morning. It will be ok. I'm running dotnet restore on each repo before merging

**Pull request merged**